django_project/geocontext/models/context_service_registry.py

```diff
@@ -307,3 +332,35 @@ def build_query_url(self, x, y, srid=4326):
             url += '&BBOX=' + bbox_string
 
             return url
+        # For the ESRI ArcREST
+        # standard a URL is
+        # constructed as with
+        # the WFS standard.
+        else:
+            if self.query_type == \
+                    ContextServiceRegistry.ARCREST:
+                if srid != self.srid:
+                    x, y = convert_coordinate(x, y, srid, self.srid)
+                bbox = get_bbox(x, y)
+                bbox_string = ','.join([str(i) for i in bbox])
+
+            parameters = {
+
+                'f': 'json',
+                'geometryType': 'esriGeometryPoint',
+                'geometry': '{x:' + str(x) + ', y:' + str(y) + '}',
```

Comment:
Please read up on python string interpolation....while you can do this, it is not idiomatic python to do it this way....